fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Analyzer.java

```diff
@@ -121,6 +124,12 @@ private static List<RewriteJob> buildAnalyzeJobs(Optional<CustomTableResolver> c
             bottomUp(new CheckAnalysis()),
             topDown(new EliminateGroupByConstant()),
             topDown(new NormalizeAggregate()),
+            bottomUp(new SemiJoinCommute()),
+            bottomUp(
+                    new CollectSubQueryAlias(),
+                    new CollectJoinConstraint()
+            ),
+            topDown(new LeadingJoin()),
```

Comment:
Pls move this rule from rewrite stage to analyze stage.

Reply:
done